cwms-data-api/src/test/java/cwms/cda/data/dao/location/kind/EmbankmentDaoIT.java

```diff
@@ -55,19 +58,21 @@
 import static org.junit.jupiter.api.Assertions.*;
 
 @Tag("integration")
+@TestInstance(Lifecycle.PER_CLASS)
 final class EmbankmentDaoIT extends DataApiTestIT {
-
-    private static final Location PROJECT_LOC = buildProjectLocation("PROJECT1");
-    private static final Location PROJECT_LOC2 = buildProjectLocation("PROJECT2");
-    private static final Location EMBANK_LOC1 = buildEmbankmentLocation("PROJECT-EMBANK_LOC1");
-    private static final Location EMBANK_LOC2 = buildEmbankmentLocation("EMBANK_LOC2");
-    private static final Location EMBANK_LOC3 = buildEmbankmentLocation("EMBANK_LOC3");
+    
+    private static final String PROJECT_OFFICE_ID = "SPK";
+    private static final Location PROJECT_LOC = buildProjectLocation("PROJECT1emdao");
+    private static final Location PROJECT_LOC2 = buildProjectLocation("PROJECT2emdao");
+    private static final Location EMBANK_LOC1 = buildEmbankmentLocation("PROJECT-EMBANK_LOC1dao");
+    private static final Location EMBANK_LOC2 = buildEmbankmentLocation("EMBANK_LOC2dao");
+    private static final Location EMBANK_LOC3 = buildEmbankmentLocation("EMBANK_LOC3dao");
 
     @BeforeAll
-    public static void setup() throws Exception {
+    public void setup() throws Exception {
```

Comment:
Pretty sure BeforeAll requires static

Reply:
Look at the annotation added to the class. That's only true if the test lifecycle isn't set to "one instance of the class".
Might not have been necessary here as the trackers in DataApiTest are static, but I forgot that while I was experimenting.

Reply:
Oh interesting!  I didn't know that, thank you for pointing that out.  I haven't used the TestInstance annotation before, so I'll have to look into it, and see how I can incorporate it in my work.